Advanced/Play Asset Delivery/Assets/PlayAssetDelivery/Runtime/CustomAssetPackUtility.cs

```diff
@@ -33,7 +36,58 @@ public enum DeliveryType
 
     public class CustomAssetPackUtility
     {
+        public const string kBuildFolderName = "Build";
+        public const string kPackContentFolderName = "CustomAssetPackContent";
+
+        public const string kBuildProcessorDataFilename = "BuildProcessorData.json";
+        public const string kRuntimeDataFilename = "CustomAssetPacksData.json";
+
+        public static string RootDirectory
+        {
+            get { return Path.Combine("Assets", "PlayAssetDelivery"); }
+        }
+
+        public static string BuildRootDirectory
+        {
+            get { return Path.Combine(RootDirectory, kBuildFolderName); }
+        }
+
+        public static string PackContentRootDirectory
+        {
+            get { return Path.Combine(BuildRootDirectory, kPackContentFolderName); }
+        }
+
+        public static string BuildProcessorDataPath
+        {
+            get { return Path.Combine(BuildRootDirectory, kBuildProcessorDataFilename); }
+        }
+
+        public static string CustomAssetPacksDataEditorPath
+        {
+            get { return Path.Combine(BuildRootDirectory, kRuntimeDataFilename); }
```

Comment:
If this could be used with Editor api's like AssetDatabase, then it'll be best to avoid Path.Combine. On windows it will add \\ between the two strings, Unity API's will expect /